actions/generate-schedule/users/usersource.go

```diff
@@ -0,0 +1,52 @@
+package users
+
+import (
+	"container/list"
+	"fmt"
+	"sort"
+)
+
+// Source represents a source of usernames for rotation
+type Source interface {
+	StartAfter(user string) error
+	NextUser() string
+}
+
+// StaticSource is a base implementation of static list of usernames.
+type StaticSource struct {
```

Comment:
Does this need to be exported?  Could `NewStaticSource` return the interface type?

Reply:
I suppose so, but it's intended to be a constructor. I plan on using it from the `github` package when the GH Teams impl is added.

Reply:
Oh, so the idea is that the GH Teams implementation will itslef use a `StaticSource` behind the scenes?  If so, then I see why this is public.

Reply:
Correct. The GH impl will query for team names, then feed them into StaticeSource